### ISSUE TITLE
Just wondering if you could recreate the results of the original paper?

### ISSUE THREAD

**Issue by Manikvsin** [RBirkeland/MVCNN-PyTorch#6]
Hi,

I was just wondering if you had any data on the accuracy of the MVCNN when using Alexnet and Resnet?

Thanks,
Manik

**Comment by RBirkeland** [comment]
Already asked in #5, but since then I've tested and reached a validation accuracy of 90% using resnet34
